lib/gitlab/ci/templates/jobs/gradle/Test.yml

```diff
@@ -1,11 +1,16 @@
+include:
+  - remote: https://raw.githubusercontent.com/chesapeaketechnology/gitlab-templates/get-custom-gradle-working-and-move-chmod-before-script/lib/gitlab/ci/templates/references/gradle/GradleJavaBase.yml
```

Comment:
Move this from get-custom-gradle-working-and-move-chmod-before-script to release/4 before merging.